demo/src/MemoryModelsUserInput.tsx

```diff
@@ -112,24 +98,17 @@ function MemoryModelsTextInput(props: MemoryModelsTextInputPropTypes) {
     };
 
     return (
-        <CardContent>
-            <TextField
-                id="multiline-memory-models-textfield"
-                data-testid="textfield-input"
-                label="Enter memory model JSON here"
-                multiline
-                fullWidth
-                rows={10}
-                variant="outlined"
-                value={props.textData}
-                onChange={handleTextFieldChange}
-                style={{
-                    width: "100%",
-                    height: "80%",
-                    fontFamily: "Monospace",
```

Comment:
You should keep the `Monospace` font here, that's for the JSON input

Reply:
We put the Monospace font back in the style component, but for some reason it's no longer rendering on the demo website (it doesn't render on the live version either). Would you like us to keep it for now and come back to it later?

Reply:
Ah okay great, indeed this isn't applying in the way we'd expect. I'm okay for you to *delete* the `style` prop here entirely (as you originally had it) and we can actually apply the font later.